### PR TITLE
Documentation: Rename viper to vyper

### PR DESCRIPTION
**Pull request by jacqueswww**
### - What I did

Part of #501.

- Renamed binary to vyper.
- Renamed all occurences of viper to vyper.

### - How I did it

make html

### - How to verify it

Make the docs and browser through them.

### - Description for the changelog
Renamed all occurences of viper to vyper in the documentation.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ Following the principles and goals, Viper **does not** provide the following fea
 Compatibility-breaking Changelog
 ********************************
 
-* **2017.12.25**: Change name from Viper to Vyper
+* **2017.12.25**: Change name from Vyper to Vyper
```
[the remark this answers is not in the document]
Comment (jacqueswww) [reply docs/index.rst]: True, I got a bit carried away perhaps ;) shoyld I remove it completely or just state that the command was renamed to vyper?

[the remark this answers is not in the document]
Comment (fubuloubu) [reply docs/index.rst]: Keep the statement that the rename occuring, this is a major breaking change

[the remark this answers is not in the document]
Comment (mattermat) [reply docs/index.rst]: I'm agreed with fubuloubu! :)

**Comment by mattermat** [comment]
Furthermore, you have to change also the occurrencies in [setup.py](https://github.com/ethereum/vyper/blob/master/setup.py#L32) in order to pass CI tests.

**Comment by jacqueswww** [comment]
@mattermat will do. Since I am renaming it here, I'll make the same changes here. Also the module needs renaming - will do that next ;)

**Comment by fubuloubu** [comment]
@DavidKnott needs a `pre beta` tag

**Comment by DavidKnott** [comment]
@jacqueswww Rebase and I'm happy to merge this 😄

**Comment by jacqueswww** [comment]
@DavidKnott updated.

**Comment by fubuloubu** [comment]
Woot! Is this task totally complete now?

**Pull request merged**